### ISSUE TITLE
Errror running mantika/logstash-dynamodb-streams

### ISSUE THREAD

**Issue by shalomvolchok** [mantika/dockerfiles#2]
I built the latest image, but when I run it I get this error... am I missing something?

```
The error reported is: 
you might need to reinstall the gem which depends on the missing jar or in case there is Jars.lock then resolve the jars with `lock_jars` command

no such file to load -- com/amazonaws/aws-java-sdk-elasticbeanstalk/1.10.11/aws-java-sdk-elasticbeanstalk-1.10.11 (LoadError)
```

**Comment by marcosnils** [comment]
@shalomvolchok thx for the notification. I'll check and come back.

**Comment by marcosnils** [comment]
@shalomvolchok I'm not seeing that issue. Can you please provide how you're running the container and which conf you're using?

Thanks!

**Comment by shalomvolchok** [comment]
@marcosnils thanks for the quick response and for sharing this docker container. I built and ran it locally on Ubuntu, let me give it another try on a fresh EC2 instance. I will do it first thing in the morning and let you know my results.

**Comment by marcosnils** [comment]
@shalomvolchok I believe I found the issue you encountered. I've pushed a new version of the image as well as updates the corresponding dockerfile. Please give it a try and let me know your results.

**Comment by shalomvolchok** [comment]
@marcosnils Awesome!! Yes, the new version works! 

Just to document for anyone else coming here, I also needed to add `protocol => http` to the elasticsearch output config or I got an error `WARN: org.elasticsearch.bootstrap: JNA not found. native methods will be disabled.` My final working elasticsearch output config:

```
elasticsearch { 
    protocol => http
    host => "ELASTICSEARCH_IP"
    port => 9200 
}
```

I really appreciate your help!
